Modules/gcmodule.c

```diff
@@ -2271,10 +2262,6 @@ PyObject_IS_GC(PyObject *obj)
 void
 _Py_ScheduleGC(PyInterpreterState *interp)
 {
-    GCState *gcstate = &interp->gc;
```

Comment:
fyi: `_Py_ScheduleGC` not public and only called from _PyObject_GC_Link, which already checks `gcstate->collecting`.